package.json

```diff
@@ -128,12 +129,14 @@
     "jest": "^29.7.0",
     "js-yaml": "^4.1.0",
     "lint-staged": "^15.5.0",
+    "memfs": "^4.17.0",
     "prettier": "^2.8.8",
     "rimraf": "^5.0.10",
     "typescript": "~5.5.4",
     "typescript-eslint": "^8.27.0"
   },
   "engines": {
     "node": "^14.15.0 || ^16.10.0 || ^18.0.0 || >=20.0.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
why is this being set to yarn?

Reply:
ah mistake while running commands locally 😄